CHANGELOG.md

```diff
@@ -24,11 +24,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Admin LTE 2.0 used. Assets files changed.
-- Move ``sonata_wrapper`` block on ```standard_layout.html.twig```
+- Move ``sonata_wrapper`` block on ``standard_layout.html.twig``
```

Comment:
> Sorry to interrupt but: `*.rst` requires 2 backticks and `*.md` requires 1 backtick.

@javiereguiluz Just for curiosity, why `*.rst` needs two backtick?

> We've published them on symfony.com

Good! Please not branching is changed. We have 2.x, 3.x and master now. :+1: